@here/olp-sdk-dataservice-read/lib/cache/ApiCacheRepository.ts

```diff
@@ -20,9 +20,7 @@
 import { HRN } from "../HRN";
 import { KeyValueCache } from "./KeyValueCache";
 
-/**
- * The list of API names of endpoints exists in the API
- */
+// The list of API endpoints to which the Open Location Platform (OLP) SDK for Typescript has access.
```

Comment:
The list names of API endpoints to which the Open Location Platform (OLP) SDK for Typescript uses.

Reply:
Fixed